src/preset.ts

```diff
@@ -691,6 +691,8 @@ export async function expandConfigurePreset(folder: string, name: string, worksp
         }
     }
 
+    // TODO: We should try to find visual studio installations here.
```

Comment:
Why can't we do this in this PR? Is the change too risky?

Reply:
Oh, we do do this in this PR, I just forgot to remove the TODO, we do it right before returning.